docs/packages/pkg/utf8.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    utf8
```

Comment:
Should be just `utf`:
```
packages/pkg/utf8.rst:11:utf:["UT", "UHF", "Ute", "UT's", "Ctrl", "GCC", "IDE", "OSX", "RPC", "SDK", "isn", "iOS", "multi"]
writing output... [100%] user-guides/regular                                    
Spelling checker messages written to /home/travis/build/ruslo/hunter/docs/_spelling/output.txt
```
See CI testing:
* https://travis-ci.org/ruslo/hunter/builds/480211380#L2508-L2510

Reply:
fixed (I think)